extension/src/sidebar.js

```diff
@@ -344,7 +371,9 @@ async function* _bindSidebarData(response: Visits) {
     const sidebar = new Sidebar(opts);
 
     const cont = await sidebar.getContainer();
-    await sidebar.clear(); // TODO probably, unnecessary?
+    const filtersEl = await sidebar.getFiltersEl();
+    safeSetInnerHTML(filtersEl, '');
```

Comment:
em... it doesn't clean ((

Reply:
ah, that's odd! I think we should add cleaning to `safeSetInnerHTML`, must have missed it by accident

Reply:
how?

Reply:
looks like JS flow is complaining, but I think it's just because `replaceChildren` was added relatively recently, in 2020
https://developer.mozilla.org/en-US/docs/Web/API/Element#browser_compatibility
I think it should be fine just to suppress its error in this case, 2020 is quite old in browser versions